### ISSUE TITLE
Add label org.opencontainers.image.source

### ISSUE THREAD

**Issue by marwin1991** [carlossg/docker-maven#324]
Hello, for tools like renovate it is important to have define label org.opencontainers.image.source

So can we add label org.opencontainers.image.source="https://github.com/apache/maven" to DOCKERFILE ?

Base on this label, renovate will provide information about changes.

https://docs.renovatebot.com/modules/datasource/docker/
https://github.com/opencontainers/image-spec/blob/main/annotations.md

The base idea behind this label is to point to repo with dockerfile, but this repo does not use tag's or releases.

What do you think?

**Comment by carlossg** [comment]
yes, this would be useful, however because the image is created in a separate repo I don't know if it is going to work as expected
The source should be this repo https://github.com/carlossg/docker-maven
Maybe for it to work we would need to copy the maven CHANGELOG in here

**Comment by marwin1991** [comment]
@carlossg thank you so much for your replay.

> Maybe for it to work we would need to copy the maven CHANGELOG in here

How would you want to achive this?

**Comment by carlossg** [comment]
don't know, open to PRs
